integrationTests/testConsensusNode.go

```diff
@@ -92,7 +92,16 @@ func NewTestConsensusNode(
 		ShardCoordinator: shardCoordinator,
 		MultiSigner:      multiSigner,
 	}
-	tcn.initNode(consensusSize, roundTime, consensusType, eligibleMap, waitingMap, keyGen, nodeKeys.HandledKeys)
+	tcn.initNode(
+		consensusSize,
+		roundTime,
+		consensusType,
+		eligibleMap,
```

Comment:
this code is dangerous since the `eligibleMap` and `waitingMap` have the same type and one can put them in reverse order, while the compiler won't complain about it.